scripts/ganache.sh

```diff
@@ -12,10 +12,10 @@ start() {
 
 	if [ -z "${ORIGARGS[1]}" ]
 	then
-		docker run --publish 8545:8545 trufflesuite/ganache:latest -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123" --networkId 1337
+		docker run --publish 8545:8545 trufflesuite/ganache:v7.7.7 -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123" --networkId 1337
 	else
 		echo "Starting ganache ..."
-		docker run --detach --publish 8545:8545 trufflesuite/ganache:latest -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123" --networkId 1337
+		docker run --detach --publish 8545:8545 trufflesuite/ganache-cli:v7.7.7 -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123" --networkId 1337
```

Comment:
@avkos I posted a PR https://github.com/web3/web3.js/pull/6019 for ganache latest version fix, it seems due to EIP3860. Instead of fixing to v7.7.7 lets use latest with 6019 pr.